im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,651 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE" xmlns="http://purl.org/net/xbiblio/csl">
+  <info>
+    <title>Im Gespräch - Hefte der Martin Buber-Gesellschaft (German)</title>
+    <title-short>Im Gespräch</title-short>
+    <id>http://www.zotero.org/styles/im-gespraech-buber-gesellschaft</id>
+    <link href="http://www.zotero.org/styles/im-gespraech-buber-gesellschaft" rel="self"/>
+    <author>
+      <name>Michael Gorki</name>
+      <email>michael.gorki@gmx.de</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="humanities"/>
+    <issn>1616-6094</issn>
+    <updated>2015-09-01T13:28:08+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="de">
```

Comment:
why are you defining all of these terms in the style? They're part of CSL, you only need to redefine the ones that differ from https://github.com/citation-style-language/locales/blob/master/locales-de-DE.xml